ram/hram.asm

```diff
@@ -85,10 +85,10 @@ hSpriteScreenX:: db
 hSpriteScreenY:: db
 
 NEXTU
-hFF8F:: db
-hFF90:: db
-hFF91:: db
-hFF92:: db
+hSpriteCollisionsLoopCounter:: db
```

Comment:
This is used as a 0-15 offset into `SpriteCollisionBitTable` for the sprite being collision-tested with the "current sprite" (whose corresponding offset is already named `hCurrentSpriteOffset`).
```suggestion
hCollidingSpriteOffset:: db
```